input/blog/2021-11-26-cake-v2.0.0-rc0002-released.md

```diff
@@ -0,0 +1,121 @@
+﻿---
+title: Cake v2.0.0 RC 2 released
+category: Release Notes
+author: augustoproiete
+releaseName: 2.0.0-rc0002
+---
+
+Version **2.0.0 RC 2** of Cake has been released. Take it for a spin and give us feedback on our [discussion board](https://github.com/cake-build/cake/discussions/3718).
+
+This is the second (and likely last) pre-release of Cake 2.0🎉
+
+See also:
+
+- [Cake v2.0.0-rc0001 release notes](/blog/2021/11/cake-v2.0.0-rc0001-released)
+
+In this blog post, we will go through what's new in Cake RC 2, compared to [RC 1](/blog/2021/11/cake-v2.0.0-rc0001-released).
+
+### Highlights of this release
+
+- Added support for .NET v6.0.100 RTM
+- Improvements and bug fixes to the new `dotnet` aliases
+- Updated .NET target frameworks
+- Impro
```

Comment:
This seems a bit cut off?

Reply:
Thanks & fixed, meant to be `Improvements to IntelliSense support when using Visual Studio Code` (re: reference assemblies)